examples/async_transcribe_media_from_url.js

```diff
@@ -11,9 +11,20 @@ const token = require('./config/config.json').access_token;
     console.log(`Account: ${account.email}`);
     console.log(`Credits remaining: ${account.balance_seconds} seconds`);
 
+    // Configure your source media url
+    // If authorization headers are needed to access the url they can be provided as an argument, e.g.
+    // var sourceConfig  {url: 'source url', auth_headers: {"Authorization": "Bearer <token>"}});
+    var sourceConfig = {url: 'https://www.rev.ai/FTC_Sample_1.mp3', auth_headers: null};
+    // Set an optional notification url 
+    // See https://docs.rev.ai/api/asynchronous/webhooks/ for details on setting up a webhook
+    // Authorization headers url can also be added to this url, e.g. 
+    // var notificationConfig {url: 'webhook url', auth_headers: {"Authorization": "Bearer <token>"}});
+    var notificationConfig = {url: 'https://jsonplaceholder.typicode.com/posts', auth_headers: {"Authorization": "Bearer <token>"}};
```

Comment:
Might be long, let's wrap and also we should be use `const` or `let` instead of `var`
```suggestion
    const notificationConfig = {
        url: 'https://jsonplaceholder.typicode.com/posts',
        auth_headers: { "Authorization": "Bearer <token>" }
    };
```